protocol/communities/community.go

```diff
@@ -369,6 +368,10 @@ func (o *Community) MarshalJSONWithMediaServer(mediaServer *server.MediaServer)
 				HideIfPermissionsNotMet: c.HideIfPermissionsNotMet,
 				Position:                int(c.Position),
 			}
+
```

Comment:
was added in https://github.com/status-im/status-go/commit/2bc2099d55407af2c56bebe849ffed12929761ca

working on updating tests and other places with `json.Marshal(community)`

Reply:
fixed community_test.go